pkg/types/types.go

```diff
@@ -14,6 +15,7 @@ const (
 	DefaultLeaderLeaseDuration = 1500
 	DefaultLeaderRenewDeadline = 1000
 	DefaultLeaderRetryPeriod   = 500
+	AddDelTimelimit            = 2 * time.Minute
```

Comment:
Could you help me understand why 2 minutes ? 

Writing the justification in the commit msg would probably be helpful for future us.